app/src/main/java/org/thoughtcrime/securesms/giph/ui/GiphyActivity.java

```diff
@@ -59,7 +60,7 @@ private void initializeToolbar() {
     GiphyActivityToolbar toolbar = ViewUtil.findById(this, R.id.giphy_toolbar);
     toolbar.setOnFilterChangedListener(this);
     toolbar.setOnLayoutChangedListener(this);
-    toolbar.setPersistence(GiphyActivityToolbarTextSecurePreferencesPersistence.fromContext(this));
+    toolbar.setPersistence(new GiphyActivityToolbarTextSecurePreferencesPersistence());
```

Comment:
Should we make this a static class and just access it without the `new`?

Reply:
It's more correct to instantiate, as it needs `prefs`. I could make this thing get injected itself... and inject its members...